forem/forem
# Refactor: Move onboarding routes out of UsersController

## What type of PR is this? (check all applicable)
- Refactor

## Description
Reduce the file size of UsersController and increase the clarity of what the onboarding portion of it does. 

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
Try running through the onboarding flow normally (ie, fresh account sign up) and it should not run through any problem

## Added tests?
- No, the existing specs are enough

## Added to documentation?
- no documentation is needed

## [optional] Are there any post-deployment tasks we need to perform?
n/a
